lib/registration.pm

```diff
@@ -191,6 +193,42 @@ sub register_product {
     assert_script_run 'SUSEConnect -r ' . get_required_var('SCC_REGCODE');
 }
 
+=head2 register_product_proxyscc
+ 
+    register_product_proxyscc();
+
+Wrapper for SUSEConnect --url <scc_proxy_url> -r <regcode>.
+Requires SCC_URL and SCC_REGCODE variable.
+=cut
+sub register_product_proxyscc {
+    assert_script_run 'SUSEConnect --url ' . get_required_var('SCC_URL') . ' -r ' . get_required_var('SCC_REGCODE');
+}
+
+sub register_addons_text {
+    my ($addonlist) = @_;
+    $addonlist //= get_var('SCC_ADDONS');
```

Comment:
if ($myaddons ne '') {
            register_addons_text($myaddons);
########################################
Already do judgment, why still need the default value?

Reply:
This 'register_addons_text' is a common function, I will use it for patch_sle too. Then I can use the default value to the parameter, if set the default value.